base/changes.txt

```diff
@@ -6,6 +6,12 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2023-06-16  Yukai Chou <muzimuzhi@gmail.com>
+
+    * lthooks.dtx (subsection{Setting rules for hooks code}):
+    Sync \@expl@@@initialize@all@@ with \__hook_initialize_all: if latter is
+    rolled back (gh1088)
+
```

Comment:
@josephwright I saw in https://github.com/latex3/latex2e/commit/ecaf6809fde2dcabcd31d1c2080f9070a2ac2866 the changes entry is added after 
```
================================================================================
All changes above are only part of the development branch for the next release.
================================================================================
```
Should I adjust position here?

Reply:
Patch changes should be below that line, yes